src/pages/storage/StoragePools.tsx

```diff
@@ -86,19 +94,19 @@ const StoragePools: FC = () => {
               {pool.name}
             </Link>
           ),
-          role: "rowheader",
+          role: "cell",
           "aria-label": "Name",
         },
         {
           content: pool.driver,
-          role: "rowheader",
+          role: "cell",
           "aria-label": "Driver",
         },
         {
           content: <StoragePoolSize pool={pool} hasMeterBar />,
-          role: "rowheader",
+          role: "cell",
           "aria-label": "Size",
-          className: "size",
+          className: classNames("size", { clustered: isClustered }),
```

Comment:
Please adjust the layout, so the cluster names appear like a column in the storage pool list next to the size. It can be all within this cell, but the member label should be placed be next to the meter.